component-runtime-manager/src/test/java/org/talend/sdk/component/runtime/manager/service/InternalizationServiceTest.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (C) 2006-2024 Talend Inc. - www.talend.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.talend.sdk.component.runtime.manager.service;
+
+import static java.util.Collections.emptyMap;
+import static java.util.Optional.ofNullable;
+
+import java.io.File;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.talend.sdk.component.runtime.input.Mapper;
+import org.talend.sdk.component.runtime.manager.ComponentManager;
+import org.talend.sdk.component.runtime.manager.asm.PluginGenerator;
+
+public class InternalizationServiceTest {
+
+    private final PluginGenerator pluginGenerator = new PluginGenerator();
+
+    @Test
+    void wrongCL(@TempDir final File temporaryFolder) {
```

Comment:
Where is assert? 
What is testing here? 
If problem - an exception will be thrown, and if not - than it's ok, smth like that?